admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
```

Comment:
@bfallonf Not just because, hence "if there's a worthwhile way". Not a big deal, just thought I'd mention.